x-pack/plugins/cases/common/schema/index.ts

```diff
@@ -154,6 +157,33 @@ export const limitedNumberSchema = ({ fieldName, min, max }: LimitedSchemaType)
     rt.identity
   );
 
+export const limitedJSNumberSchema = ({ fieldName }: LimitedJSNumberSchemaType) =>
+  new rt.Type<number, number, unknown>(
+    'LimitedNumber',
+    rt.number.is,
+    (input, context) =>
+      either.chain(rt.number.validate(input, context), (s) => {
+        if (Math.abs(s) > Number.MAX_VALUE) {
+          return rt.failure(
+            input,
+            context,
+            `Absolute value of the ${fieldName} field cannot be more than ${Number.MAX_VALUE}.`
+          );
+        }
+
+        if (Math.abs(s) < Number.MIN_VALUE) {
+          return rt.failure(
+            input,
+            context,
+            `Absolute value of the ${fieldName} field cannot be less than ${Number.MIN_VALUE}.`
+          );
+        }
+
+        return rt.success(s);
+      }),
+    rt.identity
+  );
+
```

Comment:
I know the error message would be slightly different but couldn't we use `limitedNumberSchema` to build `limitedJSNumberSchema`?

Something like:

```suggestion
export const limitedJSNumberSchema = ({ fieldName }: LimitedJSNumberSchemaType) =>
    limitedNumberSchema({
        fieldName,
        min: Number.MIN_VALUE,
        max: Number.MAX_VALUE,
    });
```

Reply:
You are right. I also feel this way. I'll think how to do it.